src/docs/development/add-to-app/android/add-flutter-fragment.md

```diff
@@ -255,6 +291,20 @@ Please see the [instructions for instantiating and starting a `FlutterEngine`].
 To use a pre-warmed `FlutterEngine` in a `FlutterFragment`, instantiate a
 `FlutterFragment` with the `withCachedEngine()` factory method.
 
+<!-- Start code sample -->
+<ul class="nav nav-tabs" id="use-prewarmed-engine-language" role="tablist">
+  <li class="nav-item">
+    <a class="nav-link active" id="use-prewarmed-engine-java" href="#use-prewarmed-engine-java-tab" role="tab" aria-controls="use-prewarmed-engine-java" aria-selected="true">Java</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="use-prewarmed-engine-kotlin" href="#use-prewarmed-engine-kotlin-tab" role="tab" aria-controls="use-prewarmed-engine-kotlin" aria-selected="false">Kotlin</a>
+  </li>
+</ul>
+
+<div class="tab-content">
+
+<div class="tab-pane active" id="use-prewarmed-engine-java-tab" role="tabpanel" aria-labelledby="use-prewarmed-engine-java-tab" markdown="1">
+<?code-excerpt "MyApplication.java" title?>
 ```java
 // Somewhere in your app before your FlutterFragment is needed...
```

Comment:
since you called the thing MyApplication.java, you might as well reference it in this sentence.

Reply:
Done